src/manual/multiple-peerconnections/js/main.js

```diff
@@ -0,0 +1,103 @@
+
+/*
+ *  Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+ */
+
+'use strict';
+
+function logError(err) {
+  console.log(err);
+}
+
+var nPeerConnections = 10;
+var testTable = document.getElementById('test-table');
+
+function addVideoPairToTable(localView, remoteView) {
+  var newRow = testTable.insertRow(-1);
+  var localCell = newRow.insertCell(-1);
+  localCell.appendChild(localView);
+  var remoteCell = newRow.insertCell(-1);
+  remoteCell.appendChild(remoteView);
+}
+
+function PeerConnection(id) {
+  this.id = id;
+
+  this.localConnection = null;
+  this.remoteConnection = null;
+
+  this.localView = document.createElement('video');
+  this.localView.autoplay = true;
+  this.remoteView = document.createElement('video');
+  this.remoteView.autoplay = true;
+  addVideoPairToTable(this.localView, this.remoteView);
+
+  this.start = function() {
+    var onGetUserMediaSuccess = this.onGetUserMediaSuccess.bind(this);
+    navigator.mediaDevices.getUserMedia({
+      audio: true,
+      video: true
+    })
+    .then(onGetUserMediaSuccess)
+    .catch(logError);
+  }
+
+  this.onGetUserMediaSuccess = function(stream) {
+    // Display stream in local video tag.
+    this.localView.srcObject = stream;
+
+    // Create local peer connection.
+    this.localConnection = new RTCPeerConnection(null);
+    this.localConnection.onicecandidate = (event) => {
```

Comment:
jscs linter is not updated to support ES6 yet (tests are red https://travis-ci.org/webrtc/testrtc/jobs/171105141#L1121), I've planned to replace it with eslint for a while now like I've done for the other webrtc repos but have not gotten around to it yet.

In the meantime, maybe you could try changing https://github.com/webrtc/testrtc/blob/master/package.json#L26 to 2.0.0 since newer versions of jscs apparently support ES6.